src/containers/UserSites.tsx

```diff
@@ -1,15 +1,40 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import { removeUserSite, setActiveUserSite } from '../actions/point'
+import { removeUserSite, setActiveUserSite, setUserSiteLabel } from '../actions/point'
 import { UserSite as UserSiteType } from '../reducers/runConfiguration'
 
 type UserSitesProps = {
   userSites: any[]
   removeSite: (index: number) => any
   setActiveSite: (index: number | null) => any
+  setUserSiteLabel: (label: string, index: number) => any
 }
 
-const UserSite = ({ userSites, removeSite, setActiveSite }: UserSitesProps) => {
+// custom input element gets focus when mounted
+const NameInput = ({label, index, setUserSiteLabel, setIsEditingName}: {label: string, index: number, setUserSiteLabel: (label: string, index: number) => any, setIsEditingName: (nameIndex: number | null) => any }) => {
```

Comment:
This line seems long; have you run eslint?

Reply:
Whoops, nope